lib/src/main/java/grpcbridge/rpc/RpcCall.java

```diff
@@ -46,14 +46,23 @@ public RpcCall(
      */
     public ListenableFuture<RpcMessage> execute(@Nullable TracingSpan tracingSpan) {
         SettableFuture<RpcMessage> result = SettableFuture.create();
+        ServerCall.Listener<Message> listener;
+
         if (tracingSpan != null) {
             tracingSpan.attachTo(result);
+            listener = tracingSpan.callInContext(() -> method
+                    .getServerCallHandler()
+                    .startCall(
+                            new AsyncCall(method.getMethodDescriptor(), result),
+                            request.getMetadata()));
+        } else {
```

Comment:
Should we be creating a new span here?

Reply:
Not sure, are you saying we should create a span if the user hasn't provided one?

Reply:
Right, so that we see this in all the tools even if client is not providing the ID.

Reply:
If the client hasn't provided the ID we'll still create a tracing span with a random trace ID. I think the only reason the span is nullable here is because previously RpcCall.execute didn't accept a span and, technically it's a public method. But I don't think it's ever actually null when we call it from Bridge. We can probably just require a TracingSpan